apps/expo/app.config.ts

```diff
@@ -28,11 +28,11 @@ const defineConfig = (): ExpoConfig => ({
       backgroundColor: "#1F104A",
     },
   },
-  // extra: {
-  //   eas: {
-  //     projectId: "your-eas-project-id",
-  //   },
-  // },
+  extra: {
+    eas: {
+      projectId: "7712a44c-fe67-40d6-a8d4-428e2192af27",
+    },
```

Comment:
maybe this should be an environment variable? unfortunately we can't all join the same project for expo :(